build.gradle.kts

```
@@ -16,7 +17,9 @@ repositories {
 }
 
 dependencies {
-    implementation("com.github.kryonitelabs:messaging:1.2.1")
+    implementation("com.github.kryonitelabs:kryo-messaging:1.3.0")
+    implementation("com.github.kryonitelabs:kryocloud-messages:feature~update-server-command-message-SNAPSHOT")
```

Change to release 0.1.0
```suggestion
    implementation("com.github.kryonitelabs:kryocloud-messages:0.1.0")
```

---

Unfortunately, I cant add a comment to the Junit dependencies. Please update them a swell and close both Dependabot PRs.

Newest Junit version is `5.8.1`.